### PR TITLE
WIP - Split Vocab and Shapes.

### PR DESCRIPTION
**Pull request by jadelkhoury**
This is an attempt at https://github.com/oslc-op/oslc-specs/issues/374.

But I can't get the shapes to be visible in hte shapes.html document.

@berezovskyi @ndjc tips?

### CODE REVIEW

File: specs/rm/requirements-management-shapes.html [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+      <p>
+        Requirements Management servers MAY augment this vocabulary with additional classes, properties, and
+        individuals.
+      </p>
+    </section>
+
+    <section id="resourceDefinitions">
+      <h2>Requirements Management Resource Constraints</h2>
+
+      <section id="requirement">
```
Comment (berezovskyi) [inline specs/rm/requirements-management-shapes.html:220]: id="RequirementShape"

File: specs/rm/requirements-management-shapes.html [hunk 2]
```diff
@@ -0,0 +1,372 @@
...
+          Any <strong>resource</strong> asserted to be of <code>rdf:type</code>
+          <code>http://open-services.net/ns/rm#Requirement</code> MUST conform to the constraints and meaning of
+          properties defined below. Notice that partial representations of a requirement resource are admitted by this
+          specification (for example, in query results, or where <code>oslc.properties</code> has been used), and such
+          partial representations will in general not conform to these constraints.
+        </p>
+
+        <div
+          title="Shape for oslc_rm:Requirement"
+          data-include="./requirements-management-shapes.ttl#Requirement"
```
Comment (berezovskyi) [inline specs/rm/requirements-management-shapes.html:239]: `./requirements-management-shapes.ttl#RequirementShape`

File: specs/rm/requirements-management-shapes.html [hunk 3]
```diff
@@ -0,0 +1,372 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="description"
+      content="This specification defines a vocabulary and resource shapes for the Requirements Management domain."
+    />
+    <title>OSLC Requirements Management Version 2.1. Part 2: Vocabulary</title>
```
Comment (berezovskyi) [inline specs/rm/requirements-management-shapes.html:9]: Part 3

File: specs/rm/requirements-management-shapes.html [hunk 4]
```diff
@@ -0,0 +1,372 @@
...
+            title: "OSLC Requirements Management Version 2.1. Part 2: Vocabulary",
+            href: "requirements-management-vocab.html",
+          },
+          {
+            title: "OSLC Requirements Management Version 2.1. Part 3: Constraints (this document)",
+            href: "requirements-management-shapes.html",
+          },
+          {
+            title: "OSLC Requirements Management Version 2.1. Part 4: Machine-readable Vocabulary Terms",
+            href: "Requirements-management-vocab.ttl",
```
Comment (berezovskyi) [inline specs/rm/requirements-management-shapes.html:60]: lowercase R

File: specs/rm/requirements-management-shapes.html [hunk 5]
```diff
@@ -0,0 +1,372 @@
...
+          },
+          {
+            title: "OSLC Requirements Management Version 2.1. Part 5: Machine-readable Constraints",
+            href: "Requirements-management-shapes.ttl",
```
Comment (berezovskyi) [inline specs/rm/requirements-management-shapes.html:64]: lowercase R

File: specs/rm/requirements-management-shapes.html [hunk 6]
```diff
@@ -0,0 +1,372 @@
...
+          OSLCCore3: {
+            title: "OSLC Core 3.0",
+            href: "https://docs.oasis-open.org/oslc-core/oslc-core/v3.0/oslc-core-v3.0-part1-overview.html",
+            authors: ["Jim Amsden", "S. Speicher"],
+            status: "Project Specification",
+            publisher: "OASIS",
+          },
+          OSLCCore3LinkGuidance: {
+            title: "OSLC Core 3.0: Link Guidance",
+            href: "https://raw.githack.com/oslc-op/oslc-specs/master/notes/link-guidance.html",
```
Comment (berezovskyi) [inline specs/rm/requirements-management-shapes.html:92]: we shall stop using any GitHack refs, that was a stopgap measure

Comment (jadelkhoury) [reply specs/rm/requirements-management-shapes.html:92]: SO should I just have a relative URL?

**Comment by jadelkhoury** [comment]
@berezovskyi What you think. Can we merge this too? Approved?

**Pull request merged**